packages/ui-extensions/src/surfaces/admin/components/CustomerSegmentTemplate/CustomerSegmentTemplate.ts

```diff
@@ -7,20 +7,32 @@ import {createRemoteComponent} from '@remote-ui/core';
 type CustomerStandardMetafieldDependency = 'facts.birth_date';
 
 export interface CustomerSegmentTemplateProps {
-  /* Localized title of the template. */
+  /**
+   * Localized title of the template.
+   */
   title: string;
-  /* Localized description(s) of the template. */
+  /**
+   * Localized description(s) of the template.
+   */
   description: string | string[];
-  /* Code snippet to render in the template with syntax highlighting. */
+  /**
+   * Code snippet to render in the template with syntax highlighting.
+   */
   query: string;
-  /* Code snippet to insert in the segment editor. If missing, `templateQuery` will be used. */
+  /**
+   * Code snippet to insert in the segment editor. If missing, `templateQuery` will be used.
```

Comment:
where is `templateQuery` set?

```suggestion
   * The code snippet to insert in the segment editor. If no code snippet is present, then `templateQuery` is used.
```